R/write-docker.R

```diff
@@ -52,13 +55,16 @@ DEFAULT_RSPM <-  "https://packagemanager.rstudio.com"
 vetiver_write_docker <- function(vetiver_model,
                                  plumber_file = "plumber.R",
                                  path = ".",
+                                 ...,
```

Comment:
I should have added some dots before now in this function (to force users to name arguments after the dots) but I don't think it's too late to put them here.